blockcypher/api.py

```diff
@@ -1585,7 +1585,7 @@ def broadcast_signed_transaction(unsigned_tx, signatures, pubkeys, coin_symbol='
 
 
 def simple_spend(from_privkey, to_address, to_satoshis, change_address=None,
-        privkey_is_compressed=True, min_confirmations=0, api_key=None, coin_symbol='btc'):
+        privkey_is_compressed=True, min_confirmations=0, api_key=None, coin_symbol='btc', fees=None, preference='low'):
```

Comment:
Thanks for the PR. However here you are changing the default preference from `high` to `low`. See here https://github.com/blockcypher/blockcypher-python/blob/4e906391667f5d9569059cf107532b39f5624284/blockcypher/api.py#L1296.